templates/tfengine/recipes/folder.hcl

```diff
@@ -32,6 +32,13 @@ schema = {
       type    = "string"
       pattern = "^[0-9]{8,25}$"
     }
+    add_parent_folder_dependency = {
+      description = <<EOF
+        Whether to automatically add dependency on parent folder.
```

Comment:
Maybe add a sentence suggesting when you would/wouldn't want to do this.

Reply:
Good idea, done